src/blocks/stars.css

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+
+  height: 16px;
+  width: 16px;
+}
+
+.stars--1 .stars__star:nth-child(-n+4) {
```

Comment:
reuse stars styles from previous task to make it flexible

Reply:
I have only one block and the elements in it, I can’t understand how I can give these elements flexibility without additional blocks (((( Can you please help

Reply:
stars will be changed by changing modifiers, no need to write all
it is enough to write just one block (with stars--4 modifier by default)